CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
```

Comment:
The Command+K icon in the search field has the wrong ratio:


| main | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/72874ffe-5ac9-4bb9-8da7-e78fc4965ee9) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/7aeea1db-85bd-4b96-be95-c8ca4e532113) |

The command symbol should roughly be the same size as the K